### ISSUE TITLE
Add ratio of weights:updates to TensorBoard

### ISSUE THREAD

**Issue by cn4750** [LeelaChessZero/lczero-training#42]
See [here](http://cs231n.github.io/neural-networks-3/#ratio) for a description.
It seems that Minigo has [already added it](https://github.com/tensorflow/minigo/pull/63/files) to their TensorBoard.
This would help us determine when to drop the learning rate greatly.
